acct.bionimbus.org/manifest.json

```diff
@@ -49,12 +45,11 @@
     "logs_bucket": "logs-accountprod-gen3",
     "sync_from_dbgap": "False",
     "dispatcher_job_num": "10",
-    "useryaml_s3path": "s3://cdis-gen3-users/accountprod/user.yaml",
-    "netpolicy": "on"
```

Comment:
why was this deleted? Is it important?

Reply:
i mean r53

Reply:
It removed a comma (`,`) from the line with `useryaml_s3path `, which is fine.
But that next line `"netpolicy": "on"` should stay for security purposes.
(again, we can fix all that in `gitops-qa/qa-acct.planx-pla.net/manifest.json` and run the automation again to produce a new PR with all these corrections.